### ISSUE TITLE
TheOneProbe information displays correct mod name

### ISSUE THREAD

**Issue by GreatestFool** [henkelmax/fake-blocks#4]
**Bug description**
The One Probe information displays which name the fake block is from, regardless of which block it is mimicking.

**Expected behavior**
In the gif on the mod page the mod name changes to whichever mod it's from, in the gif it was from Fake Blocks to Minecraft.

When I try to replicate this behavior, mine does not change its name.

**Versions**
- Minecraft: v1.16.5
- Forge: v36.1.10
- Mod: v1.0.2

**Other mods**
I have a few other mods, but I doubt they cause this issue.
- The One Probe: v3.1.4
- Item Zoom: v2.3.0
- Just Enough Items: v7.7.0.99
- KubeJS: v1605.3.10-build.8

It still happens even if all other mods than The One Probe and Fake Blocks are disabled.

**Screenshots (Optional)**
![image](https://user-images.githubusercontent.com/3003484/119683690-23a4f980-be44-11eb-95df-37d482c70c0a.png)

**Comment by henkelmax** [comment]
This mod currently only supports WAILA.
I have to look into it, if it is possible to do the same with the one probe.

**Comment by GreatestFool** [comment]
Aha. That explains it. Was not aware it was only for WAILA, sorry for the inconvenience. 

I'd greatly appreciate if you could look into if the same is possible with The One Proble :)

**Comment by GreatestFool** [comment]
Is it [Hwyla](https://www.curseforge.com/minecraft/mc-mods/hwyla) Fake Blocks has integration with on Minecraft versions such as 1.16.5? As far as I can see, that one is archived due to TehNut not being interested in modding anymore.

If it is, an alternative to Hwyla is the fork of it called [Jade](https://www.curseforge.com/minecraft/mc-mods/jade). Though, that might be the one your mod has integration with for all I know :)

Also, sorry for messaging so quickly after the other.

---

***edit:***
![image](https://user-images.githubusercontent.com/3003484/119687687-b1361880-be47-11eb-8ffc-9e6398420693.png)

Right, looks like it doesn't work with Jade. I'm assuming that is due to the [internal namespace](https://github.com/Snownee/Jade/tree/1.16/src/main/java) of it is both waila (like the Hwyla fork also had) _as well as_ its own jade namespace?

**Comment by henkelmax** [comment]
You can use the 1.16.2 version of HWYLA with 1.16.5

**Comment by GreatestFool** [comment]
True, it does still work with that one, but is it not better to use an updated and maintained fork that includes pull requests and bug fixes instead of an archived version?

**Comment by henkelmax** [comment]
I will switch to jade if HWYLA doesn't get updated to 1.17.

**Comment by GreatestFool** [comment]
Considering the dev has publicly added to the mod page that he's no longer interested in modding, archived all of his repositories and even edited the information on his GitHub account to mention that it is a graveyard and all but abandoned it completely, I'd say that is only a matter of time :) Jade also has a 1.17 update in the works.

**Comment by henkelmax** [comment]
Then I will add jade support for 1.17

**Comment by GreatestFool** [comment]
Thank you :)

Also, #5 is not the same issue as this. I had a brainfart and forgot to change parts of the template I copied over. I've fixed that, but it'd be nice if you could re-open it and take a look at why grass does not colorize when faked.

**Comment by henkelmax** [comment]
Please open another issue for this.

**Comment by GreatestFool** [comment]
It'd be far easier to simply re-open the issue, considering it'll be a complete copy-paste of issue #5... but alright.
